Atomic/atomic.py

```diff
@@ -134,7 +134,7 @@ class Atomic(object):
                 "--name", "${NAME}",
                 "${IMAGE}"]
 
-    def __init__(self):
+    def __init__(self, is_dbus = False):
         self.d = get_docker_client()
```

Comment:
No reason to pass this in, flag should be something along the lines of 
self.args.tty=False